lib/server.js

```diff
@@ -137,7 +137,10 @@ module.exports = async config => {
             buildClusterFactory: config.buildClusterFactory,
             ecosystem: config.ecosystem,
             release: config.release,
-            queueWebhook: config.queueWebhook
+            queueWebhook: config.queueWebhook,
+            feature: {
+                unzipArtifacts: config.unzipArtifactsEnabled
```

Comment:
should we need to add `config.unzipArtifactsEnabled` to jsdoc?

Reply:
thx.
I Fixed it.